build.gradle

```diff
@@ -1,6 +1,28 @@
-group 'multi-int-store'
-version '0.1.0-SNAPSHOT'
+plugins {
+	id 'org.springframework.boot' version '2.1.4.RELEASE'
+	id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+group = 'com.connexta'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
```

Comment:
✏️ Should the java versions look like this instead?
```
    sourceCompatibility = 1.11
    targetCompatibility = 1.11
```

Reply:
Source = Language level java version
Target = bytecode java version.
I think we need both. I wasn't aware of the distinction

Reply:
I misread that. Yes, it should be 1.11